packages/mint-components/src/components/tax-and-cash/sqm-docusign-form/sqm-docusign-form-view.tsx

```diff
@@ -263,95 +266,43 @@ export const DocusignFormView = (props: DocusignFormViewProps) => {
       )}
       {/* 
       // @ts-ignore */}
```

Comment:
```suggestion
      {/* 
      // @ts-ignore */}
```
not sure if this is necessary